man/rmd/bake_dependent_roles.Rmd

```diff
@@ -0,0 +1,55 @@
+## Effects of non-standard roles
+
+
+Recipes steps can keep column(s) of your data set that should not be treated as outcomes or predictors. For example, a unique identifier column or some other ancillary data that can be used to troubleshoot issues during model development. 
+
+As an example, you might have a date column that is transformed into features using [step_date()] and.or [step_holiday()]. 
+
+```{r, results = "hide", message = FALSE, warning = FALSE}
+  library(tidymodels)
+  tidymodels_prefer()
+
+  data(Chicago, package = "modeldata")
+  chi_rec <- recipe(ridership ~ ., data = Chicago) %>% 
+    # Since `date` is symbolically on the right-hand side of the formula, it
+    # is treated as a predictor (i.e., has a role of "predictor")
+    step_holiday(date) %>% 
+    step_date(date)
+```
+
+At this pont, `date` is kept in the dataset as-is, and will be treated as a predictor (and will probably be converted to an integer column by the modeling function). 
+
+One option is to remove date since it is not directly used as a model term. [step_rm()] is a good way to accomplish this. 
+
+```{r}
+  chi_rec_rm <- chi_rec %>% step_rm(date)
+```
+
+Another approach, especially helpful during model development, is to change the role of the column to some other value via [update_role()]. 
+
+```{r}
+  chi_rec_role <- chi_rec %>% update_role(date, new_role = "date")
+```
+
+In this case, the column is still available in the data and will be carried through the analysis (e.g. during resampling). This can be helpful since, it there is a poor prediction, this column may be helpful in diagnosing the problem. 
```

Comment:
```suggestion
In this case, the column is still available in the data and will be carried through the analysis (e.g. during resampling). This can be helpful since, if there is a poor prediction, this column may be helpful in diagnosing the problem. 
```